### DISCUSSION TITLE
[Proposal] cherry-pick from GPDB

### DISCUSSION THREAD

**Discussion by roseduan** [apache/cloudberry#661] [Proposal]
### Proposal Status

Under Discussion

### Abstract

cherry-pick from GPDB

### Motivation

I am going to cherry-pick some features/bugfix/PRs from GPDB into CloudberryDB, if you want to bring some GPDB features into CloudberryDB, please let me know and I will give it priority.

### Implementation

let`s discuss.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by roseduan** [comment]
This cloud be a long term proposal, any comments will be considered in our  cherry-pick work in the future.

**Reply by tuhaihe** [reply]
yes, thanks @roseduan for creating this discussion. Now Max and I are working on drafting one strategy cherry-pick proposal from Greenplum to Cloudberry. Welcome to have your feedback on the new proposal once it's ready.

**Reply by tuhaihe** [reply]
Hi, the strategy proposal is out - #675

**Comment by roseduan** [comment]
https://github.com/apache/cloudberry/discussions/675
